tests/iprep-11-isset/scirius-categories.txt

```diff
@@ -0,0 +1,10 @@
+1,2520000,ET TOR Known Tor Exit Node Traffic
```

Comment:
Do we need the big scirius-iprep.list file ? Instead of a simple one with only the relevant IP